push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/APNsPushNotificationSender.java

```diff
@@ -208,10 +208,10 @@ private Date createFutureDateBasedOnTTL(int ttl) {
 
         // no TTL was specified on the payload, we use the MAX Default from the APNs library:
         if (ttl == -1) {
-            return new Date(System.currentTimeMillis() + EnhancedApnsNotification.MAXIMUM_EXPIRY * 1000L);
+            return new Date(EnhancedApnsNotification.MAXIMUM_EXPIRY * 1000L);
```

Comment:
`EnhancedApnsNotification.MAXIMUM_EXPIRY` is  `Integer.MAX_VALUE` [ref](https://github.com/notnoop/java-apns/blob/598b396e93221ffb7373c95ec760c2fec07601a9/src/main/java/com/notnoop/apns/EnhancedApnsNotification.java#L57) maybe we should use: `new Date(Long.MAX_VALUE)` that way it's the maximum date value?